docs/blog/newsletter-december-2022.rst

```diff
@@ -0,0 +1,121 @@
+.. post:: December 07, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2022
+#########################################
+
+Hello to all the lovely documentarians out there. The end of the year is approaching, which is a wonderful time to reflect on all the things we've done in 2022. And on what to look forward to for next year. I hope your personal list of accomplishments is long and your list of remaining things to do isn't quite as daunting as mine.
```

Comment:
what do you think of addressing the reader directly here? 

> Hello lovely documentarians.

maybe that is _too_ familiar?

Reply:
And I'd opt for 

> And to reflect on what to look forward to for next year.

or similar